x-pack/plugins/security_solution/server/usage/detections/detection_rule_helpers.ts

```diff
@@ -121,6 +159,18 @@ export const updateDetectionRuleUsage = (
         disabled: !detectionRuleMetric.enabled ? usage.query.disabled + 1 : usage.query.disabled,
         alerts: usage.query.alerts + detectionRuleMetric.alert_count_daily,
         cases: usage.query.cases + detectionRuleMetric.cases_count_total,
+        legacy_notifications_enabled: legacyNotificationEnabled
```

Comment:
Apologies, my brain may be a bit scrambled right now, but here we are updating the field totals. A rule and action relationship is one to many right? Can a rule have numerous actions in which case the addition could be larger than `+ 1`?

I think I'm thinking of the question differently, but you're answering what the tickets asked, so 👍🏽

Reply:
>A rule and action relationship is one to many right? 

Correct. I am not tracking the "total actions", but rather how many rules have at least one action on it and if that rule is enabled or not.